### ISSUE TITLE
Using another BLE usb adapter

### ISSUE THREAD

**Issue by alphaomicron** [evilsocket/bleah#34]
Supposing we want to use another BLE usb adapter for longer range shouldn't this work under command? : bleah -i

**Comment by WitcherDev** [comment]
yes you provide an index in -i sth like `bleah -i X t0`  ,where X comes from hciconfig
